app/database/queries/breeze_buddy/users.py

```
@@ -0,0 +1,490 @@
+                row["merchant_ids"]
+                if isinstance(row["merchant_ids"], list)
+                else json.loads(row["merchant_ids"])
+            )
+
+            # Return None for wildcard (means all merchants)
+            if "*" in merchant_ids:
+                return None
+
+            return merchant_ids
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)